src/pymanopt/manifolds/product.py

```diff
@@ -73,19 +70,11 @@ def __init__(
 
         dimension = np.sum([manifold.dim for manifold in manifolds])
         point_layout = tuple(manifold.point_layout for manifold in manifolds)
-        # TODO: set a backend to be able to accss
         super().__init__(
             name,
             dimension,
             point_layout=point_layout,
-            backend=manifolds[0].backend,
-        )
-
-    @Manifold.backend.setter
-    def backend(self, backend: Backend):
-        warnings.warn(
-            "Setting backend of Product manifold is not supported. "
-            "One should directly set the backend of each underlying manifold."
+            backend=manifolds[0].backend.to_real_backend(),
```

Comment:
add a comment to say real doesn't matter here (only used to instantiate the backend of a function)